crates/holochain/tests/signals/mod.rs

```diff
@@ -0,0 +1,135 @@
+//! Tests for local and remote signals using rendezvous config
+//!
+
+use fallible_iterator::FallibleIterator;
+use futures::StreamExt;
+use hdk::prelude::ExternIO;
+use holochain::sweettest::{SweetCell, SweetConductorBatch, SweetConductorConfig, SweetDnaFile};
+use holochain_types::signal::Signal;
+use holochain_wasm_test_utils::TestWasm;
+use holochain_zome_types::RemoteSignal;
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+
+#[derive(Serialize, Deserialize, Debug)]
+struct SignalMessage {
+    value: String,
+}
+
+fn to_signal_message(signal: Signal) -> SignalMessage {
+    match signal {
+        Signal::App { signal, .. } => signal.into_inner().decode().unwrap(),
+        _ => {
+            panic!("Only expected app signals");
+        }
+    }
+}
+
+#[tokio::test(flavor = "multi_thread")]
+#[cfg(feature = "slow_tests")]
+async fn remote_signals_batch() -> anyhow::Result<()> {
+    holochain_trace::test_run().ok();
+
+    let mut conductors =
+        SweetConductorBatch::from_config_rendezvous(2, SweetConductorConfig::rendezvous()).await;
```

Comment:
I didn't know we have this in sweetest!

Reply:
Yes, and it's opt-in so most of our tests are still using tx2. A topic for discussion soon I think